pylint/checkers/typecheck.py

```diff
@@ -974,6 +978,12 @@ def visit_call(self, node):
                 self.add_message('missing-kwoa', node=node,
                                  args=(name, callable_name))
 
+        # Check for any keyword parameter before variable length parameters
+        # (*args)
+        if called.args.vararg and called.args.defaults:
+            self.add_message('keyword-arg-before-vararg', node=node,
+                             args=(callable_name, node.func.as_string()))
```

Comment:
Let's just get the name of the function without calling ``as_string()``. I think ``called`` is the callable in this context, you could try to get it's name by checking if it's an Attribute or a Name node and retrieving their corresponding name attribute.